Assets/Mirror/Examples/Chat/Scripts/ChatNetworkManager.cs

```diff
@@ -46,7 +46,7 @@ private void OnCreatePlayer(NetworkConnection connection, CreatePlayerMessage cr
             playergo.GetComponent<Player>().playerName = createPlayerMessage.name;
 
             // set it as the player
-            NetworkServer.AddPlayerForConnection(connection, playergo);
+            NetworkServer.AddPlayerForConnection(connection, client, playergo);
```

Comment:
Do we need the connection?

Reply:
Perhaps I'm wrong, but what I understood from my tests is that we need it for OnServerAddPlayer. If we get the NetworkConnection from client.connection, we end up with a NetworkConnectionToServer instead of a NetworkConnectionToClient.

Reply:
ohhhh,   I see,  and why do we need the client here?  this method is supposed to be server side no?

Reply:
For some reason, it doesn't need it but only in host-mode case. Perhaps a Mirror bug?